src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -270,20 +292,43 @@ public function blockForm($form, FormStateInterface $form_state) {
         '#maxlength' => 2048,
       ];
 
-      $form['exclude_by_location'] = $base_by_location + [
+      $form['location_category'] = [
```

Comment:
This new grouping will change the path to settings in the form, but since we're still saving them in a flat array (see `$conf`) we don't need any update hooks or changes in the config schema.